site/content/docs/user/quick-start.md

```diff
@@ -34,18 +34,21 @@ Stable releases are generally recommended for CI usage in particular.
 To install, download the binary for your platform from "Assets" and place this
 into your `$PATH`.
 
-On macOS / Linux:
+On Linux:
 
 {{< codeFromInline lang="bash" >}}
 curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-$(uname)-amd64
```

Comment:
might as well drop this uname then?

I think this is an unintended side-effect of the download alias, github was serving these case-insensitive.

Reply:
Your call: IMO at some point this will work again, if we rename or alias the download aliases , so I think its good to keep the generic functionality.

Reply:
I don't think there's a good way to rename this, all of the naming is generic, and I think splitting it up is probably cleaner anyhow, so we can list all the options for a platform.

IIRC on some systems uname returns `Linux` as well ...?

Reply:
I think that removing `uname` is more consistent with the rest of the doc now

Reply:
done